openlibrary/catalog/works/find_works.py

```diff
@@ -535,8 +535,6 @@ def new_work(akey, w, do_updates, fh_log):
         try:
             print(ol.save_many(update, "add editions to new work"), file=fh_log)
         except:
```

Comment:
This needs to raise!

Reply:
Or remove the try and except lines and dedent line 536.  Bare exceptions blow in any case.  
https://realpython.com/the-most-diabolical-python-antipattern/

Reply:
That does make more sense, thanks for the tip.